app-editors/retext/retext-7.2.0-r1.ebuild

```diff
@@ -7,8 +7,10 @@ EAPI=7
 # old compat matrix is no longer accessible as of 2021-02-13 but stated back
 # in 2020-07-05 that PyQt5 was explicitly not supported.
 PYTHON_COMPAT=( python3_{8,9} )
+# See https://dev.gentoo.org/~mgorny/python-guide/distutils.html
+DISTUTILS_USE_SETUPTOOLS=bdepend
```

Comment:
This is the default-defined value, it doesn't need to be put into the ebuild.

Reply:
fixed locally. Had the idea from https://bugs.gentoo.org/746071.

> ```
> To avoid breakage for existing installations the deps are not made optional
> but moved into USE flags (bug 772197).
> ```
> 
> If they are **optional**, then there is no breakage. Just put them into `optfeature`.
OK. I actually had to put markdown support back into the package unconditionally - sorry for that noise.